internal/cli/cmd/install.go

```diff
@@ -60,7 +60,7 @@ cilium install --context kind-cluster1 --cluster-id 1 --cluster-name cluster1
 	cmd.Flags().StringVarP(&params.Namespace, "namespace", "n", "kube-system", "Namespace to install Cilium into")
 	cmd.Flags().StringVar(&params.ClusterName, "cluster-name", "", "Name of the cluster")
 	cmd.Flags().StringSliceVar(&params.DisableChecks, "disable-check", []string{}, "Disable a particular validation check")
-	cmd.Flags().StringVar(&params.Version, "version", "", "Cilium version to install")
+	cmd.Flags().StringVar(&params.Version, "version", defaults.Version, "Cilium version to install")
```

Comment:
I think this would mention the current default version in the help output. If not, can we add that?

Reply:
Yes, the help output show the default version after this patch.

Before the patch:
```console
% go run ./cmd/cilium install --help
...
      --version string                  Cilium version to install
...
```

After the patch:
```console
% go run ./cmd/cilium install --help
...
      --version string                  Cilium version to install (default "v1.9.7")
...
```